packages/origin-247-certificate/test/setup.ts

```diff
@@ -119,7 +126,9 @@ export const bootstrapTestInstance: any = async () => {
 
     const connection = await app.resolve<Connection>(getConnectionToken());
     const tables = connection.entityMetadatas
-        .filter((e) => e.tableName !== 'issuer_blockchain_properties' && e.tableName !== 'issuer_signer')
+        .filter(
+            (e) => e.tableName !== 'issuer_blockchain_properties' && e.tableName !== 'issuer_signer'
```

Comment:
I think this doesn't apply anymore and needs to use new table name, right?

Reply:
As long as we still have issuer-api dependency, it applies.

Reply:
Oh right, just remember to remove it later